velox/exec/RowsStreamingWindowBuild.cpp

```diff
@@ -122,21 +123,19 @@ void RowsStreamingWindowBuild::noMoreInput() {
 }
 
 std::shared_ptr<WindowPartition> RowsStreamingWindowBuild::nextPartition() {
-  return outputPartition();
-}
-
-bool RowsStreamingWindowBuild::hasNextPartition() {
-  // Note: an exception may be thrown if remove the windowPartition_ in
-  // nextPartition method. This is because hasNextPartition might return true,
-  // but when we actually retrieve the window partition in nextPartition, it
-  // could be empty after the removal.
+  outputPartition_++;
```

Comment:
Should outputPartition_++ be inside the while loop ? I generally feel the while loop is not needed, but if we are writing that way, then its more logical to increment outputPartition_ in while loop.